app/integration/project-generator/src/main/resources/io/syndesis/integration/project/generator/templates/application.properties.mustache

```diff
@@ -0,0 +1,6 @@
+# Logging
+logging.level.io.syndesis = DEBUG
```

Comment:
Should this be hardcoded or would it make more sense to make it configurable ? // @paoloantinori this migh affect the support page, too.

Reply:
At the moment it is hardcoded but yeas it make sense to make it configurable. My idea is to add some logging option to the integration so that this can be controlled (i.e. you may ahve a flag on the integration page to turn on/of debugging). Or we can use an environment variable but that would mean to change the build config.